arlo-client/src/components/MultiJurisdictionAudit/StatusBox/index.test.tsx

```diff
@@ -0,0 +1,370 @@
+      )
+      screen.getByText('Audit setup is not complete.')
+      screen.getByText('The audit has not started.')
+      screen.getByText('1 of 2 jurisdictions have completed file uploads.')
+    })
+
+    it('renders full uploads state', () => {
+      render(
+        <Router>
+          <AuditAdminStatusBox
+            rounds={[]}
+            jurisdictions={jurisdictionMocks.twoProcessed}
+            contests={[]}
+            auditSettings={auditSettings.blank}
+          />
+        </Router>
+      )
+      screen.getByText('Audit setup is not complete.')
+      screen.getByText('The audit has not started.')
+      screen.getByText('2 of 2 jurisdictions have completed file uploads.')
+    })
+
+    it('renders finished setup state', () => {
+      render(
+        <Router>
+          <AuditAdminStatusBox
+            rounds={[]}
+            jurisdictions={jurisdictionMocks.twoProcessed}
+            contests={contestMocks.filledTargeted.contests}
+            auditSettings={auditSettings.all}
+          />
+        </Router>
+      )
+      screen.getByText('Audit setup is complete.')
+      screen.getByText('The audit has not started.')
+      screen.getByText('2 of 2 jurisdictions have completed file uploads.')
+    })
+
+    it('renders one of two jurisdictions done round one state', () => {
+      render(
+        <Router>
+          <AuditAdminStatusBox
+            rounds={roundMocks.singleIncomplete}
+            jurisdictions={jurisdictionMocks.oneComplete}
+            contests={contestMocks.filledTargeted.contests}
+            auditSettings={auditSettings.all}
+          />
+        </Router>
+      )
+      screen.getByText('Round 1 of the audit is in progress')
+      screen.getByText('1 of 2 jurisdictions have completed Round 1')
+    })
+
+    it('renders round complete, need another round state', () => {
+      render(
+        <Router>
+          <AuditAdminStatusBox
+            rounds={roundMocks.needAnother}
+            jurisdictions={jurisdictionMocks.twoComplete}
+            contests={contestMocks.filledTargeted.contests}
+            auditSettings={auditSettings.all}
+          />
+        </Router>
+      )
+      screen.getByText(
+        'Round 1 of the audit is complete - another round is needed'
+      )
+      screen.getByText('When you are ready, start Round 2')
+      screen.getByText('Start Round 2')
+    })
+
+    it('creates the next round', () => {
+      apiMock.mockImplementation(async () => ({
+        message: 'success',
+        ok: true,
+      }))
+      render(
+        <Router>
+          <AuditAdminStatusBox
+            rounds={roundMocks.needAnother}
+            jurisdictions={jurisdictionMocks.twoComplete}
+            contests={contestMocks.filledTargeted.contests}
+            auditSettings={auditSettings.all}
+          />
+        </Router>
+      )
+      fireEvent.click(screen.getByRole('button', { name: 'Start Round 2' }), {
+        bubbles: true,
+      })
```

Comment:
i started using `userEvent` in place of `fireEvent`: https://github.com/testing-library/user-event

Reply:
@jonahkagan would you recommend I install it here and use it, or should we do a separate PR to convert everything over to `userEvent` instead of `fireEvent`?

Reply:
Good q. No strong opinion! I think either way would work

Reply:
I think I'd prefer a separate PR so this one doesn't get too much scope creep, haha.